rocAL/rocAL/include/pipeline/commons.h

```diff
@@ -112,7 +112,19 @@ enum class RocalROIType
 
 typedef struct
 {
-    unsigned x1, y1, x2, y2;
+    unsigned l, t, r, b;
+} RocalRoiLtrb;
+
+typedef struct
+{
+    unsigned x, y;
+    unsigned roiWidth, roiHeight;
+} RocalRoiXywh;
```

Comment:
This need not be RocalRoiXywh

Remove Rocal here

Reply:
Done